Lib9c/TableData/ArenaSheet.cs

```diff
@@ -52,9 +62,19 @@ public bool IsTheRoundOpened(long blockIndex)
         public class Row : SheetRow<int>
         {
             public override int Key => Id;
+
             public int Id { get; private set; }
+            
+            /// <summary>
+            /// The [`Round`] contains 8 elements always
+            /// </summary>
             public List<RoundData> Round { get; private set; }
 
+            /// <summary>
+            /// Championship number equals with Id always
+            /// </summary>
+            public int ChampionshipNumber => Id;
```

Comment:
Because they share the same value, this may be redundant information.

I think using ArenaSheet.Row.Id as return value for ChampionshipNumber has the same function as we use ItemUsable.ItemId as return value for INonFungibleItem.NonFungibleId or ITradableItem.TradableId.

Even if the summary of ArenaSheet.Row.Id contains enough information, I don't think the user will want to find the Championship number by the name of Id. Just like when we want to get the Item's NonFungibleId, we don't reference it as ItemUsable.ItemId.